src/svgbob.rs

```diff
@@ -16,5 +16,5 @@ pub fn bob_handler(s: &str, settings: &Settings) -> String {
 	svg.render_with_indent(&mut source, 0, true).expect("html render");
 
 	let style = Style::new("svg { width: 100% !important; }").set("type", "text/css");
-	format!("<div style='width:100%; height:{}px;'>{}{}</div>", height, style, source).replace('\n', "")
+	format!("<div style='width:100%; height:{}px; fill:var(--fg);'>{}{}</div>", height, style, source).replace('\n', "")
```

Comment:
I just noticed that there is a number of [configuration options](https://github.com/boozook/mdbook-svgbob#config) — could it be that we just need to update the default for `fill_color`?

Reply:
Updating the default for `fill_color` does not work for some reason.

Reply:
Okay, thanks for checking! Then I just think you need to rebase this PR on top of `master` and that will make the tests pass.

Reply:
@boozook, do you know more about or are you fine with just adding the CSS here?